tpl/layout/svg/shoppingbag.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" id="Ebene_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 64 64" style="enable-background:new 0 0 64 64;" xml:space="preserve">
+    <style type="text/css">
+        .st0{fill: #009ebc;}
```

Comment:
Is this inline style really necessary? This should be done with css (less), too...